app/models/qa/iri_template/url_config.rb

```diff
@@ -0,0 +1,56 @@
+# Provide access to iri template configuration.
+module Qa
+  module IriTemplate
+    class UrlConfig
+      TYPE = "IriTemplate".freeze
+      CONTEXT = "http://www.w3.org/ns/hydra/context.jsonld".freeze
+      attr_reader :template # [String] the URL template with variables for substitution (required)
+      attr_reader :variable_representation # [String] always "BasicRepresentation" # TODO what other values are supported and what do they mean
+      attr_reader :mapping # [Array<Qa::IriTempalte::VariableMap>] array of maps for use with a template (required)
+
+      # @param [Hash] url_template configuration hash for the iri template
+      # @option url_template [String] :template the URL template with variables for substitution (required)
```

Comment:
@elrayle is this url_config?

Reply:
This was an error.  On the @param, it should be the parameter's name (i.e. url_config), then each of the @option lines should also start with the parameter's name.  For each option, after the type, it should have the Hash key name.

https://www.rubydoc.info/gems/yard/file/docs/Tags.md#option